code/modules/mob/living/bot/secbot.dm

```diff
@@ -24,7 +24,16 @@
 
 	var/list/threat_found_sounds = list('sound/voice/bcriminal.ogg', 'sound/voice/bjustice.ogg', 'sound/voice/bfreeze.ogg')
 	var/list/preparing_arrest_sounds = list('sound/voice/bgod.ogg', 'sound/voice/biamthelaw.ogg', 'sound/voice/bsecureday.ogg', 'sound/voice/bradio.ogg', 'sound/voice/bcreep.ogg')
-
+//VOREStation Add - They don't like being pulled
+/mob/living/bot/secbot/Life()
+	..()
+	if(health > 0 && on && pulledby)
+		if(isliving(pulledby))
+			var/mob/living/L = pulledby
+			UnarmedAttack(L)
+			say("Do not interfere with active law enforcement routines!")
```

Comment:
I demand this say, "DO NOT INTERFERE WITH SECURITY OPERATIONS."

One, it's a Fallout reference.
Two, because I said so. :3